pkg/minikube/out/out.go

```diff
@@ -108,11 +126,32 @@ func String(format string, a ...interface{}) {
 	}
 
 	klog.Infof(format, a...)
+	if spin.Active() == true {
+		spin.Stop()
+	}
+	_, err := fmt.Fprintf(outFile, format, a...)
+	if err != nil {
+		klog.Errorf("Fprintf failed: %v", err)
+	}
+}
+
+// SpinnerString writes a basic formatted string and spinner to stdout
+func SpinnerString(format string, a ...interface{}) {
```

Comment:
lets add a boolean to Step func so next person edits the step code does not forget to edit the spinning step too

Reply:
that func was removed